public/js/lib/pokedex/ui/Carousel.js

```diff
@@ -71,16 +71,32 @@ export default class Carousel extends React.PureComponent {
                     children={<DescriptionSVG/>}
                     onTouchTap={
                         () => {
-                            this.props.setSelectedPokemonForDetails(this.props.pokemons.all[key]);
+                            let pokemon = {};
+                            pokemon.current = this.props.pokemons.all[key];
+                            pokemon.evolution = [];
+                            pokemon.starter = {};
+                            //Gère les évolutions
+                            for(var i = 0; i < this.props.pokemons.all.length; i++) {
```

Comment:
for natif c'est plus rapide de 1 millionième de micro-seconde, c'est pertinent sur des tableaux de 3Mb. Avec nos 151 pov' pokemon franchement on a rien à craindre